qa/tox.ini

```diff
@@ -18,3 +18,8 @@ commands=flake8 --select=F,E9 --exclude=venv,.tox
 basepython = python3
 deps = mypy
 commands = mypy {posargs:.}
+
+[testenv:import-tasks]
+basepython = python3
+deps = {env:TEUTHOLOGY_GIT:git+https://github.com/ceph/teuthology.git@master}
```

Comment:
```suggestion
deps = {env:TEUTHOLOGY_GIT:git+https://github.com/ceph/teuthology.git@master}}#egg=teuthology[coverage,orchestra,test]
```

Reply:
thanks @toabctl! i was using this test to find the py3 issues this afternoon. and #34139 is the outcome =)

Reply:
nice! :)

Reply:
Fixed (and I dropped the barbican fix commit. You have that also fixed in https://github.com/ceph/ceph/pull/34139)